rdrf/explorer/templates/explorer/query.html

```diff
@@ -192,49 +192,45 @@
         <button id="hide-description-btn" type="button" class="btn btn-primary btn-sm">{% trans "Hide description" %}</button>
```

Comment:
Have a look at `ReportDataTableView` / `report_table_view.html` as well. The buttons generated by datatables aren't being formatted correctly.

Reply:
Can you show me how to use this functionality?

Reply:
Sure

Reply:
Seems it was a caching issue as well, but I'll take you through the interface